ViewControllers/PlayingNowCollectionViewController.swift

```diff
@@ -20,6 +20,7 @@ class PlayingNowCollectionViewController: UICollectionViewController {
         
         collectionView.register(PlayingNowCollectionViewCell.self, forCellWithReuseIdentifier: PlayingNowCollectionViewCell.reuseIdentifier)
 
+        // Shall I make it a separate method to reuse it in willDisplayCell method?
```

Comment:
Seems a good idea. Improves readability and you can reuse it anywhere that is needed